models/user.js

```diff
@@ -67,7 +67,23 @@ const userSchema = new mongoose.Schema({
   avatar : {
     type : Buffer
   },
-  role:[String]
+  role:[String],
+  department : {
+    type: String
+  },
+  timeline : [{
+    name :  String,
```

Comment:
Rename to "caption"

Also add a section "comment: String", if the user wants to comment on this section of timeline then they can.